kinesis_consumer.cpp

```diff
@@ -148,6 +149,28 @@ lookup_region(const char *r)
 	return -1;
 }
 
+static void
+parse_url(ClientConfiguration &config,
```

Comment:
I'm not sure how the AWS library will respond. I meant we should add some validation for the URL.